pkg/cmd/create/root.go

```diff
@@ -20,6 +20,7 @@ import (
 const (
 	recreateClusterUsage   = "Delete cluster first if it already exists."
 	buildNameUsage         = "Name for build (Prefix for kind cluster name, pod names, etc)."
+	devModeUsage           = "When enabled, the platform will run the core packages with developer password."
```

Comment:
1. I am just not sold on the name `dev mode`. idpbuilder is primary used for development and testing purposes already. I think we should use more explicit name with a short flag.
2. Even if we accept dev mode, we shouldn't limit the flag name to just passwords. dev mode could mean a lot of different things. This should be a meta flag.

Reply:
> 1. idpbuilder is primary used for development and testing purposes already. I think we should use more explicit name with a short flag.

If this is the case, then we should install gitea and argocd using a non generated password ;-)

Reply:
> 2\. Even if we accept dev mode, we shouldn't limit the flag name to just passwords. dev mode could mean a lot of different things. This should be a meta flag.

What about using the the parameter: `--devPwd` or `--devPassword` as boolean @nabuskey

Reply:
I am ok with `dev-pwd` or `dev-password` since we use snake case for our flags.

Reply:
Finally we aligned our paths => I will then use `--dev-mode`

Reply:
`--dev-mode` is a pretty common parameter used by many developer's tools and CLI which could also be used to support different additional features in the future. On the opposite `--static-creds` or `--static-credentials` parameters are probably too restrictive, don't help the users as they have no idea about what `static` word means